spec/rental_spec.rb

```diff
@@ -0,0 +1,47 @@
+require_relative '../student'
+require_relative '../teacher'
+require_relative '../book'
+require_relative '../rental'
+describe Rental do
+  let(:student) { Student.new(18, 'John', 1, 'Math') }
+  let(:teacher) { Teacher.new(30, 'David', 2, 'Math') }
+  let(:book) { Book.new('The Great Gatsby', 'F. Scott Fitzgerald') }
+  describe '#new' do
+    it 'should create a new rental' do
+      expect(Rental.new('2020-10-20', book, student)).to be_an_instance_of(Rental)
+    end
+  end
+  describe '#date' do
+    it 'should return the date of the rental' do
+      rental = Rental.new('2020-10-20', book, student)
```

Comment:
- [ ] I see that you're using this rental assignment in almost every spec, wouldn't it be better to extract this using the `let` keyword just as you did with student, teacher, and book? Remember to DRY! 🤓 🔧